modules/UI/toolbars/ToolbarToggler.js

```diff
@@ -35,7 +35,6 @@ function hideToolbar(force) { // eslint-disable-line no-unused-vars
     toolbarTimeoutObject = null;
 
     if (Toolbar.isHovered()
-            || APP.UI.isRingOverlayVisible()
```

Comment:
Well I was confused because at some point there was decision to display hangup button and not the toolbar.